### PR TITLE
New package: Sequeler-0.6.0

### PR DESCRIPTION
**Pull request by TeddyDD**
not as pretty as on Elementary but works
https://github.com/Alecaddd/sequeler

### CODE REVIEW

File: srcpkgs/Sequeler/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'Sequeler'
+pkgname=Sequeler
+version=0.6.0
+revision=1
+wrksrc="sequeler-${version}"
+build_style=meson
+hostmakedepends="pkg-config vala"
+makedepends="granite-devel glib-devel gtk+3-devel libgee08-devel libxml2-devel gtksourceview-devel libsecret-devel libgda-devel"
```
[the remark this answers is not in the document]
Comment (TeddyDD) [reply srcpkgs/Sequeler/template]: It requires glib-compile-resources binary to build so glib-devel is required

[the remark this answers is not in the document]
Comment (Johnnynator) [reply srcpkgs/Sequeler/template]: Then you probably need glib-devel in hostmakedepends and only libglib-devel in makedepends. Which should also fix cross compiling.

[the remark this answers is not in the document]
Comment (TeddyDD) [reply srcpkgs/Sequeler/template]: cross compiling is broken because of granite, but I'll try it

**Comment by maxice8** [comment]
On Sun, Sep 02, 2018 at 03:34:40AM -0700, Daniel Lewan wrote:
> TeddyDD commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,14 @@
> +# Template file for 'Sequeler'
> +pkgname=Sequeler
> +version=0.6.0
> +revision=1
> +wrksrc="sequeler-${version}"
> +build_style=meson
> +hostmakedepends="pkg-config vala"
> +makedepends="granite-devel glib-devel gtk+3-devel libgee08-devel libxml2-devel gtksourceview-devel libsecret-devel libgda-devel"
> 
> It requires glib-compile-resources binary to build so glib-devel is required
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/2308#discussion_r214535873

if it needs to execute a binary it should be in hostmakedepends

**Comment by TeddyDD** [comment]
added glib-devel to hostmakedepends

File: srcpkgs/Sequeler/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'Sequeler'
+pkgname=Sequeler
```
Comment (CameronNemo) [inline srcpkgs/Sequeler/template]: You could just use an uncapitalized package name to be consistent with the upstream repo and avoid repetition from having to define wrksrc manually.

File: srcpkgs/Sequeler/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+version=0.6.0
+revision=1
+wrksrc="sequeler-${version}"
+build_style=meson
+hostmakedepends="pkg-config glib-devel vala"
```
Comment (CameronNemo) [inline srcpkgs/Sequeler/template]: There is an xbps-trigger to run glib-compile-resources, the build should not do it manually when it detects DESTDIR. Looking at `meson/post_install.py`, glib-devel should not be necessary.

File: srcpkgs/Sequeler/template [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+makedepends="granite-devel libglib-devel  gtk+3-devel libgee08-devel
+libxml2-devel gtksourceview-devel libsecret-devel libgda-devel"
```
Comment (CameronNemo) [inline srcpkgs/Sequeler/template]: indent wrapped lines with a single space

File: srcpkgs/Sequeler/template (hunk 3 above)
Comment (TeddyDD) [reply srcpkgs/Sequeler/template]: well, it fails to build without it
```
FileNotFoundError: [Errno 2] No such file or directory: 'glib-compile-resources': 'glib-compile-resources'
```

Comment (CameronNemo) [reply srcpkgs/Sequeler/template]: Oh I see the meson.build uses that command. It should be okay to keep then.

**Pull request merged**